### ISSUE TITLE
TypeError: Cannot read properties of null (reading 'useMemo') in next.js

### ISSUE THREAD

**Issue by TimPchelintsev** [TypeCellOS/BlockNote#805]
**Describe the bug**
Getting  error `TypeError: Cannot read properties of null (reading 'useMemo')` on page load using latest `next.js 14.2.3`(pages router) and `@blocknote/react 0.13.5`, dynamic import same like in https://www.blocknotejs.org/docs/advanced/nextjs

**Misc**
- Node version: 18.17.1
- Package manager: yarn
- Browser: Chrome

**Comment by lequocbinh04** [comment]
Same problem

**Comment by YousefED** [comment]
Do you have the same with the latest version of BlockNote?

**Comment by jerome-geek** [comment]
Same as me.
I Think there is a problem using `yarn`, there is no erros in `npm`.

- Node version: 20.11.0
- Package manager: yarn
```
"@blocknote/core": "^0.14.5",
"@blocknote/mantine": "^0.14.6",
"@blocknote/react": "^0.14.6",
```

**Comment by Zonxpk** [comment]
solved after upgrade from 0.16 -> 0.19
